### ISSUE TITLE
Add Matomo

### ISSUE THREAD

**Issue by giovannism20** [LibreSign/site#1]
Matomo is an alternative alternative for Google Analytics. Is important to have data about the users and to get to know our audience better.

https://matomo.org/

**Comment by vitormattos** [comment]
Production tag:

```html
<!-- Matomo Tag Manager -->
<script>
  var _mtm = window._mtm = window._mtm || [];
  _mtm.push({'mtm.startTime': (new Date().getTime()), 'event': 'mtm.Start'});
  (function() {
    var d=document, g=d.createElement('script'), s=d.getElementsByTagName('script')[0];
    g.async=true; g.src='https://matomo.librecode.coop/js/container_8jNjdh8C.js'; s.parentNode.insertBefore(g,s);
  })();
</script>
<!-- End Matomo Tag Manager -->
```
Development tag:
```html
<!-- Matomo Tag Manager -->
<script>
  var _mtm = window._mtm = window._mtm || [];
  _mtm.push({'mtm.startTime': (new Date().getTime()), 'event': 'mtm.Start'});
  (function() {
    var d=document, g=d.createElement('script'), s=d.getElementsByTagName('script')[0];
    g.async=true; g.src='https://matomo.librecode.coop/js/container_8jNjdh8C_dev_dc9cf71ee2745d3690156798.js'; s.parentNode.insertBefore(g,s);
  })();
</script>
<!-- End Matomo Tag Manager -->
```

**Comment by vitormattos** [comment]
Only by reference: https://jigsaw.tighten.com/docs/building-and-previewing-environments/

We can put at config file the function getenv to get the environment value and also put at docker-compose.yml the environment that we will use, I suggest to use APP_ENV and a value as string (production or dev) to identify the current environment.

And we also will need to put a conditional at blade file of our layout to identify the environment

**Comment by vitormattos** [comment]
closed by: #17
